Cargo.toml

```diff
@@ -28,7 +28,7 @@ serialport = { version = "4", optional = true }
 
 # ledger
 regex = { version = "1.6.0", optional = true }
-ledger_bitcoin_client = { version = "0.1.0", optional = true }
+ledger_bitcoin_client = { version = "0.1.2", optional = true }
```

Comment:
Why two, and why a patch bump? Technically it's a breaking change (you add a new requirement to the trait impl). Not sure it matters a lot for now.

Reply:
It is a dep, not the crate version and it is required in order to have the ledger call.

Reply:
Wow, sorry i read it quickly and just assumed you had bumped the crate version :man_facepalming:

Reply:
bumping here:
https://github.com/wizardsardine/async-hwi/pull/20/files
I still use patch version because I wanted user to understand that everything was very very beta